Assets/Scripts/Dashboard/Tutorial/TutorialController.cs

```diff
@@ -49,23 +52,30 @@ public void Next()
         }
         else
         {
-            defaultLobbyDashboard.SetActive(true);
-            skipButton.SetActive(false);
-            gameObject.SetActive(false);
+            End();
         }
     }
 
-    private void Skip()
+    private void End()
     {
         foreach (TutorialStep tutorialStep in tutorialSteps)
         {
             tutorialStep.Cleanup(TMP);
             if (tutorialStep.AllowActiveFalse == true) tutorialStep.gameObject.SetActive(false);
         }
         skipButton.SetActive(false);
+        skipButton.GetComponent<Button>().onClick.RemoveListener(End);
+
+        replayButton.SetActive(true);
+        replayButton.GetComponent<Button>().onClick.AddListener(ReplayTutorial);
         defaultLobbyDashboard.SetActive(true);
-        gameObject.SetActive(false);
+
         _teleportController.enabled = true;
         cameraRig.transform.position = mainIslandSpawnPoint.transform.position;
     }
+
+    private void ReplayTutorial()
+    {
+        gameObject.GetComponent<SceneSwitch>().StartTransition();
```

Comment:
Don't you need to reset all of the tutorial stuff? I don't think replaying the tutorial should be reloading the scene? it should be resetting the tutorial objects to the beginning of the tutorial?